app/views/admin/divisions/index.html.slim

```diff
@@ -29,4 +30,6 @@
 
       .clearfix
 
-= render_grid(@divisions_grid)
+.index-grid class="#{'no-records' if @no_records}"
+  = render_grid(@divisions_grid)
+  = content_tag(:div, t('common.no_records_match'), class: 'alert alert-info no-records-match')
```

Comment:
This seems like a fine approach but can we move it into a helper method to avoid duplication?